geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewaySenderEventImpl.java

```diff
@@ -117,7 +117,7 @@
   /**
    * The serialized new value for this event's key. May not be computed at construction time.
    */
-  protected byte[] value;
+  protected volatile byte[] value;
```

Comment:
Did you mean to make this `volatile`? It seems unrelated.

Reply:
The double checking is done in getSerializedValue, the local variable use was present but the value was not volatile, which is recommended post jdk5